feedback/static/feedback.css

```diff
@@ -551,6 +558,145 @@ body.ios .feedback-message tr .glyphicon {
 	align-items: flex-end;
 }
 
+/* Popovers */
+.popover {
+	width: 45vw;
+	max-width: 600px;
+}
+@media (max-width: 700px) {
+	.popover {
+		width: 90vw;
+	}
+}
+.popover:has( .pos-right) {
+	left: auto !important;
+	right: 2rem !important;
+}
+
+.popover > .popover-content {
+	overflow-y: scroll;
+	max-height: 85vh;
+}
+
+/* conversation preview popovers */
+.popover .conversation-panel:not(:last-child) {
+	padding-bottom: 1rem;
+	border-bottom: 1px solid lightgray;
+	margin-bottom: 1rem;
+}
+
+.popover .conversation-panel .panel-heading {
+	margin-bottom: -3px;
+}
+.popover.feedback-message {
+	padding-bottom: 0;
+}
+.popover .feedback-message td {
+	padding: 3px;
+}
+.popover .feedback-message > .panel-footer {
+	cursor: auto;
+}
+.popover .feedback-message > .panel-footer .timestamp:after {
+	display: none;
+}
+.popover .conversation-panel .display-response {
+	color: #000;
+}
+.popover .conversation-panel .message-info {
+	height: 0.5em;
+	line-height: 1;
+}
+
+/* background questionnaire popovers */
+.background-display h3 {
+	margin-top: 4px;
+	font-size: 20px;
+}
+.popover .background-display .control-label,
+.popover .background-display .question-description p {
+	margin-bottom: 2px;
+}
+.popover .background-display .form-group {
+	margin-bottom: 0.75em;
+}
+.popover .background-display .form-group:last-child {
+	margin-bottom: 0
+}
+.popover .background-display label > input[type="radio"]:checked + span,
+.popover .background-display label > input[type="checkbox"]:checked + span {
+	background-color: #bce8f1;
+	box-shadow: 0 0 0 3px #bce8f1;
+}
+
+/* points display popovers */
+.popover:has( .points-display) {
+	left: auto !important;
+	right: 0 !important;
+	max-width: 400px;
+}
+.popover:has( .points-display) .arrow {
+	left: auto !important;
+	right: 1em !important;
+}
```

Comment:
Is it absolutely necessary to use `!important` here? It is not a good practice to use it in CSS, unless there is no other way to make the styles work properly. If there is no other (relatively easy) way to make this work, add a comment explaining why we use `!important` here. 

https://developer.mozilla.org/en-US/docs/Web/CSS/important#best_practices